tests/localization/color/analyser_test.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from mrc.localization.color.analyser import Analyser
+
+
```

Comment:
We have to add here a new - higher level of abstraction called K-Means strategy.